lib/suspenders/generators/production/deployment.rb

```diff
@@ -0,0 +1,32 @@
+require "rails/generators"
+
+module Suspenders
+  module Production
+    class DeploymentGenerator < Rails::Generators::Base
+      source_root File.expand_path(
+        File.join("..", "..", "..", "..", "templates"),
```

Comment:
Would it be helpful to have a `Suspenders::Generators::Base` that inherits from `Rails::Generators::Base` and knows where all the templates live?

Reply:
Believe it or not, I was just thinking that about while reviewing #957. I'm going to do that as a separate commit in which I refactor everything.